integrations/runllm-widget/src/script.raw.js

```diff
@@ -9,6 +9,7 @@
     const communityUrl = '<COMMUNITY_URL>';
     const communityType = '<COMMUNITY_TYPE>';
     const disableAskAPerson = '<DISABLE_ASK_A_PERSON>';
+    const nativeAiExperience = '<NATIVE_AI_EXPERIENCE>';
```

Comment:
```suggestion
    const nativeAiExperience = <NATIVE_AI_EXPERIENCE>;
```

So that you can use it as a boolean

Reply:
I don’t think this works? Getting a linting error - this is a .js file and it’s reading as a tag